### PR TITLE
Badges : Add goreportcard and coveralls community badges

### PR DESCRIPTION
**Pull request by nplanel**
Signed-off-by: Nicolas PLANEL <nplanel@redhat.com>

### CODE REVIEW

**Comment by safchain** [comment]
rerun devstack

**Comment by safchain** [comment]
rerun devstack-plugin

**Comment by nplanel** [comment]
run tests

**Comment by nplanel** [comment]
run scale-tests

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
debugging ubuntu 14.04 vm

**Comment by nplanel** [comment]
rerun tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-selenium-tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-selenium-tests

**Comment by nplanel** [comment]
skydive-scale-tests

**Comment by nplanel** [comment]
re-run tests

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-selenium-tests

**Comment by nplanel** [comment]
run skydive-unit-tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
re-run tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
ssh skydive-functional-tests-backend-orientdb

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
re-run tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
re-run tests

**Comment by nplanel** [comment]
re-run tests

**Comment by nplanel** [comment]
re-run tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
re-run tests

File: tests/coverage/coverage.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
```
Comment (safchain) [inline tests/coverage/coverage.go]: 2017

File: tests/coverage/coverage_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
```
Comment (safchain) [inline tests/coverage/coverage_test.go]: ditto

File: tests/coverage/coverage.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+package coverage
+
+import (
+	"github.com/skydive-project/skydive/cmd"
+)
+
+func skdyiveCoverage() {
+	cmd.RootCmd.Execute()
```
Comment (safchain) [inline tests/coverage/coverage.go]: this file seems to be useless ? This line could be used in the Test file

Comment (nplanel) [reply tests/coverage/coverage.go]: it's a trick, because you cannot have a package without a source file and only a test file

**Comment by nplanel** [comment]
run skydive-coverage

**Pull request merged**